content-src/experiments/universal-search.yaml

```diff
@@ -94,74 +94,111 @@ modified: '2016-07-20T15:42:28.873321Z'
-  fall: we showed fewer results, and our click-through rate was worse.
-  Intuitively this made sense. The duplicates we removed would include many
-  sites that users revisit habitually, so our previously higher click-through
-  rate included many clicks stolen from history navigation.</p>
-  <img src="/static/images/experiments/universal-search/graduation/results_2.png" />
-  <figcaption>Card types shown and selected. We added movie cards in early October.</figcaption>
+  <p><figcaption>User data showing percentage of time recommendations were displayed and 
+  percentage of time participants clicked on recommendations.</figcaption></p>
+  <p>Key events to note in the graph<br />
+  <ul><li>May to July -  Top-level domain and Wikipedia results displayed </li>
+  <li>Mid-July to Mid-August - No data collected during telemetry system upgrade</li>
+  <li>Mid-August to September - Top-level domain (de-duped) and Wikipedia results displayed</li>
+  <li>October to November - Rich movie card, top-level domain (de-duped) and Wikipedia results displayed</li></ul></p>
+  <p>Weekly Performance by Result Type</p>
+  <figure>
+  <img src="/static/images/experiments/universal-search/graduation/graph2.png" />
+  <figcaption>
+  <img style="width:36px;" src="/static/images/experiments/universal-search/graduation/graph2_key1.png" /> Top-Level Domains: Displayed<br />
+  <img style="width:36px;" src="/static/images/experiments/universal-search/graduation/graph2_key2.png" /> Top-Level Domains: Clicked<br />
+  <img style="width:36px;" src="/static/images/experiments/universal-search/graduation/graph2_key3.png" /> Wikipedia: Displayed<br />
+  <img style="width:36px;" src="/static/images/experiments/universal-search/graduation/graph2_key4.png" /> Wikipedia: Clicked<br />
+  <img style="width:36px;" src="/static/images/experiments/universal-search/graduation/graph2_key5.png" /> Movie Cards: Displayed<br />
+  <img style="width:36px;" src="/static/images/experiments/universal-search/graduation/graph2_key6.png" /> Movie Cards: Clicked<br />
+  </figcaption>
   </figure>
-  <p>In October 2016, we added a third result type, rich movie cards, which we
-  hypothesized would be shown less often but clicked more frequently than the
-  other types. A breakdown of clicks by result type showed that Wikipedia pages
-  were shown most and clicked least. Top-level domains were the most consistent
-  performer. Movie Cards had the least amount of test data, but outperformed
-  Wikipedia pages. In aggregate, our click-through-rate for search results
-  settled around 7% by the end of the experiment. In the graphs below, we have a
-  data gap from July through the first week of August where we upgraded Test
-  Pilot Telemetry.</p>
-  <img src="/static/images/experiments/universal-search/graduation/results_3.png" />
-  <figcaption>Results from one of our feedbck surveys.</figcaption>
+  <p><figcaption>Percentage of results displayed and clicked by type.</figcaption></p>
```

Comment:
https://validator.w3.org/nu/#textarea seems to dislike the `<figcaption>` tag that isn't nested within a `<figure>` tag.